save-orchestrator/src/main/kotlin/org/cqfn/save/orchestrator/service/DockerService.kt

```diff
@@ -313,8 +313,10 @@ class DockerService(private val configProperties: ConfigProperties) {
                     .resolve(it.testRootPath)
                 )
             val currentSuiteDestination = destination.resolve("$PREFIX_FOR_SUITES_LOCATION_IN_STANDARD_MODE${it.testSuiteRepoUrl.hashCode()}_${it.testRootPath.hashCode()}")
-            log.info("Copying suite ${it.name} from $standardTestSuiteAbsolutePath into $currentSuiteDestination/...")
-            copyRecursivelyWithAttributes(standardTestSuiteAbsolutePath, currentSuiteDestination)
+            if (!currentSuiteDestination.exists()) {
```

Comment:
Previously for test suites from one standard repository and with similar testRootPath we overwrite data any time, but 
at first it were extra actions (however for now our standard repos not really large, so it was not critical)
at second, damn `Files.copy` even with replace existing flag raise exceptions if it will try copy into not empty dir


[REPLACE_EXISTING – Performs the copy even when the target file already exists. If the target is a symbolic link, the link itself is copied (and not the target of the link). If the target is a non-empty directory, the copy fails with the DirectoryNotEmptyException exception.](https://www.codejava.net/java-se/file-io/java-nio-copy-file-or-directory-examples)

Reply:
Yes, I think in some way, it was the lazy solution (if some files weren't removed before new execution, by this overwrite we were be able to not care about it). However, for these reasons it's better to use some manual deletion, which I added in the latest commit

I've tested git and standard modes with reruns, also I tested different executions, which used the same directories in docker image (because of the same initial data, provided from frontend), and all worked fine without any clashes

Reply:
> `so we won't always have !exists() == true here?`

Despite the fact, that I didn't met it before review commit too, now it shouldn't happened for sure

Reply:
Ok, great